MobileApps/Ideas.js

```diff
@@ -19,4 +19,14 @@ const ideas = [
     TeckStack:[],//if applicable    
     MatrixBreaker: "true", //Here Matrix breaker means if the app will help the user to came out negative Habit patterns Like- addiction, Hatered,lust ,cling to materialism and Take him/her to better control of mind ,Innerpeace ,More awareness ,Minimilism, Warriorness ,Self realisation ,Patience and love etc.
   }
+  {
+    Name: "Food-cart", //if applicable
+    Type: "Mobile",
+    Description:
+      "This App will deliver the food ordered by you to your place as soon as possible",
+    Industry: "Online food ordering",
+    Video: "https://www.youtube.com/watch?v=9nWcPPHBzMk", //Video for reference only
+    MatrixBreaker: "false", //Here Matrix breaker means if the app will help the user to came out negative Habit patterns Like- addiction, Hatered,lust ,cling to materialism and Take him/her to better control of mind ,Innerpeace ,More awareness ,Minimilism, Warriorness ,Self realisation ,Patience and love etc.
+  }
+  
```

Comment:
Please add some new creative ideas to your app, You have to add at least one new feature or idea to your app that makes your app different from already existing food delivery apps, Then only we can accept your Pull request. Thanks for contributing.